docs/criteria/connectors/rc.md

```diff
@@ -37,33 +37,37 @@ These connectors are exempt from running the TPC derived test packages, and rely
-| Clickhouse    | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| Databricks    | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| Delta Lake    | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| Dremio        | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| File          | ✅ (1) | ✅ (0.5) | ❌ | ❌ | ✅ | ❌ |
-| FTP/SFTP      | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
-| GraphQL       | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
-| GitHub        | ❌ | ❌ | ⚠️ | ❌ | ❌ | ⚠️ |
-| HTTP/HTTPS    | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
-| MS SQL        | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| MySQL         | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| ODBC          | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| PostgreSQL    | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| Sharepoint    | ❌ | ❌ | ⚠️ | ❌ | ❌ | ⚠️ |
-| Snowflake     | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| Spice.AI      | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
-| S3            | ✅ (1) | ✅ (0.5) | ⚠️ | ❌ | ✅ | ❌ |
-| Azure BlobFS  | ✅ (1) | ✅ (0.5) | ⚠️ | ❌ | ✅ | ❌ |
-| Spark         | ✅ (1) | ✅ (0.5) | ✅ | ✅ | ✅ | ✅ |
+| Connector | [TPC-H Derived Tests (Scale Factor)](#tpc-h) | [TPC-DS Derived Tests (Scale Factor)](#tpc-ds) | [Federation](#federation) | [Data Correctness](#data-correctness) | [Streaming](#streaming) | [Native Schema Inference](#schema-inference) |
+| - | - | - | - | - | - | - |
+| Clickhouse              | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Databricks              | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Delta Lake              | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Dremio                  | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| DuckDB                  | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| File                    | ✅ (1) | ✅ (1) | ❌ | ❌ | ✅ | ❌ |
+| FTP/SFTP                | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| GraphQL                 | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| GitHub                  | ❌ | ❌ | ⚠️ | ❌ | ❌ | ⚠️ |
+| HTTP/HTTPS              | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
+| MS SQL                  | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| MySQL                   | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| ODBC                    | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| PostgreSQL              | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Sharepoint              | ❌ | ❌ | ⚠️ | ❌ | ❌ | ⚠️ |
+| Snowflake               | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| Spice.AI Cloud Platform | ✅ (100) | ✅ (100) | ✅ | ✅ | ✅ | ✅ |
+| S3                      | ✅ (1) | ✅ (1) | ⚠️ | ❌ | ✅ | ❌ |
+| Azure BlobFS            | ✅ (1) | ✅ (1) | ⚠️ | ❌ | ✅ | ❌ |
+| Spark                   | ✅ (1) | ✅ (1) | ✅ | ✅ | ✅ | ✅ |
 
 ### All Connectors
 
 These requirements are imposed on every connector, regardless of the selected features/tests in the above table.
 
-- [ ] [Core Arrow Data Types](../definitions.md) are supported
+The RC release criteria expand on and require that all [Beta release criteria](./beta.md) continue to pass for the connector.
+
+- [ ] All [Beta release criteria](./beta.md) pass for this connector.
+- [ ] [Core Arrow Data Types](../definitions.md) are supported.
```

Comment:
Should be around native types support and not Arrow type IMO. Arrow Type is just an internal representation.

Reply:
I'm not sure I agree. We support native types through Arrow types. If we can't support a native type, we can't support the Arrow type either. For accelerators, the focus is on Arrow conversion back to native (which for Arrow, is still Arrow) - accelerators don't know about the connector's native type the data originated from.

I think adding a documentation table for any mappings from native types makes sense, but I don't think we should change the supported data types.